pkg/controller/container-runtime-config/helpers.go

```diff
@@ -342,6 +342,10 @@ func getManagedKeyReg(pool *mcfgv1.MachineConfigPool, client mcfgclientset.Inter
 	return ctrlcommon.GetManagedKey(pool, client, "99", "registries", getManagedKeyRegDeprecated(pool))
 }
 
+func getManagedKeyDefaultContainerRuntime(pool *mcfgv1.MachineConfigPool) string {
+	return fmt.Sprintf("00-override-%s-generated-crio-default-container-runtime", pool.Name)
```

Comment:
There should be a comment here describing why 00-override (it needs to be between 00-default and 01-ctrcfg-defaultRuntime)

Reply:
Done!